code/provision/playbooks/roles/prometheus/files/values.yml

```diff
@@ -1,275 +1,1715 @@
-rbac:
+# Default values for prometheus-operator.
```

Comment:
Do we have to keep the full values.yml in the repo? I think for the nginx helm chart @iwalmsley worked out how to just have a values file that overrides the values we wanted to change. It makes it a lot easier to find the bits that we've configured so worth looking to see if you can do the same here I think.

Reply:
Yes, I agree. Would make it easier to see what changes was made.  Will make those changes and do a new PR.

Reply:
Hi Gary, I have updated the PR. Kindly review